examples/pipistrello/agent.py

```diff
@@ -0,0 +1,125 @@
+import asyncio
+
+from dotenv import load_dotenv
+from loguru import logger
+from typing_extensions import override
+
+from examples.pipistrello.parser import PipistrelloParser
+from examples.pipistrello.perception import PipistrelloPerception
+from examples.pipistrello.prompt import PipistrelloPrompt
+from notte.browser.observation import Observation
+from notte.common.agent.base import BaseAgent
+from notte.common.agent.config import AgentConfig
+from notte.common.agent.types import AgentOutput
+from notte.common.credentials.models import VaultInterface
+from notte.common.tools.conversation import Conversation
+from notte.common.tracer import LlmUsageDictTracer
+from notte.controller.actions import CompletionAction
+from notte.env import NotteEnv, NotteEnvConfig
+from notte.llms.engine import LLMEngine
+
+_ = load_dotenv()
+
+
+class PipistrelloAgentConfig(AgentConfig):
+    env: NotteEnvConfig = NotteEnvConfig().use_llm()
+
+
+class PipistrelloAgent(BaseAgent):
```

Comment:
now that you finalised the implement I don't think this `Pipistrello` folder is needed anymore. Can you delete that and integrate it directly in falco/gufo ?